src/conversion_helper.js

```diff
@@ -158,6 +158,13 @@ const unitLookupList = [
     "ignoredKeywords" : ukSubreddits
   },
   {
+    "imperialUnits" : [/f(?:oo|ee)?t per second/, /ft\/s(ec)?/],
```

Comment:
Can you make the first one `/f(?:oo|ee)?t per sec(ond)?/` in order to support text like `ft per sec` etc.